### PR TITLE
Fix Custom Views panel UI

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Fix Custom Views panel UI

#### Description

We realized we broke the UI of the Custom Views panel container when working in the new `Drawer` component so, instead of having a clean container with no predefined header:

Screenshot 2023-12-04 at 18 32 55

we were including a non-wanted header:

Screenshot 2023-12-04 at 18 33 42

On top of that, I took the opportunity to update `@emotion/react` dependency to match the one used in `ui-kit`.

### CODE REVIEW

File: packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx [hunk 1]
```diff
@@ -130,21 +136,24 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
   ].join('/');
 
   return (
-    <Drawer
+    <ModalPage
```
Comment (CarlosCortizasCT) [inline packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:139]: This is the file with the relevant changes.

File: packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx [hunk 2]
```diff
@@ -130,21 +136,24 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
...
       isOpen
-      title={`Custom View: ${props.customView.defaultLabel}`}
       onClose={props.onClose}
       size={panelSize === 'small' ? 10 : 30}
+      title=""
```
Comment (emmenko) [inline packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:143]: Shouldn't we provide a title here?

File: packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx [hunk 3]
```diff
@@ -130,21 +136,24 @@ function CustomViewLoader(props: TCustomViewLoaderProps) {
   ].join('/');
 
   return (
-    <Drawer
```
Comment (emmenko) [inline packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:133]: Ah so we can't use the Drawer component anymore?

**Review by emmenko — APPROVED** [review]
Thanks!

File: packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx (hunk 2 above)
Comment (CarlosCortizasCT) [reply packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:143]: The issue is that the title is required in the `ModalPage` component and I didn't want to change that but we don't want to display any title in this use case.

File: packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx (hunk 3 above)
Comment (CarlosCortizasCT) [reply packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:133]: Not in this case.
The `Drawer` component will always use a header which is something we don't want in this use case.

File: packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx (hunk 2 above)
Comment (emmenko) [reply packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:143]: The `title` of the `ModalPage` is only used for accessibility if I'm not mistaken, it's not a visible value.

Comment (CarlosCortizasCT) [reply packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:143]: So are you suggesting we should make that property optional in the `ModalPage` component?

Comment (emmenko) [reply packages/application-components/src/components/custom-views/custom-view-loader/custom-view-loader.tsx:143]: No I'm suggesting to use it 😄

**Pull request merged**